### PR TITLE
[MNT] Fix codespell config

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
This PR fixes `codespell`'s configuration and includes now the `tests` directory. It will also correct the detected spelling errors.

### CODE REVIEW

**Comment by GuillaumeFavelier** [comment]
[It works now](https://travis-ci.org/pyvista/pyvista/jobs/646785169#L717-L736):
```
pyvista/utilities/fileio.py:155: seperate  ==> separate

pyvista/utilities/errors.py:228: incase  ==> in case

pyvista/plotting/qt_plotting.py:467: droping  ==> dropping

pyvista/plotting/renderer.py:245: renderering  ==> rendering

pyvista/core/objects.py:416: te  ==> the, be

examples/02-plot/depth-peeling.py:9: defualt  ==> default

tests/test_composite.py:50: disctionaries  ==> dictionaries

tests/test_polydata.py:514: Creat  ==> Create

tests/test_common.py:571: empy  ==> empty

tests/test_qt_plotting.py:110: bottons  ==> bottoms, buttons

tests/test_qt_plotting.py:155: perfrom  ==> perform

tests/test_grid.py:319: Usign  ==> Using, unsign

tests/test_grid.py:417: mehtods  ==> methods

tests/test_objects.py:2: ojects  ==> objects

tests/test_filters.py:23: avaialble  ==> available

tests/test_filters.py:84: avaialble  ==> available

tests/test_filters.py:105: avaialble  ==> available

tests/test_filters.py:270: doesnt  ==> doesn't, does not

tests/test_plotting.py:781: arry  ==> array, carry

tests/test_plotting.py:786: transperancy  ==> transparency
```

**Comment by GuillaumeFavelier** [comment]
Sadly I have to exclude the `docs` directory because of the formatting in `*.ipynb` files, `codespell` detects `\nTo` as a spelling error.

**Comment by GuillaumeFavelier** [comment]
Feel free to merge @pyvista/developers

**Comment by akaszynski** [comment]
Noticed that as well when going through some spelling mistakes locally.  I'm looking to see if there's a fix for line endings as I think the notebooks should be spell checked as well.

**Comment by GuillaumeFavelier** [comment]
> I'm looking to see if there's a fix for line endings as I think the notebooks should be spell checked as well.

That would be awesome!

**Comment by akaszynski** [comment]
Opened an issue at https://github.com/codespell-project/codespell/issues/1421

Seems that it's not possible based on a quick google, but maybe there's a way.  You're good to merge.

**Comment by GuillaumeFavelier** [comment]
I would prefer to wait for https://github.com/pyvista/pyvista/pull/587 to be merged first just in case there are some conflicts.

**Comment by akaszynski** [comment]
> Opened an issue at [codespell-project/codespell#1421](https://github.com/codespell-project/codespell/issues/1421)
> 
> Seems that it's not possible based on a quick google, but maybe there's a way. You're good to merge.

We'll see if this gets traction:
https://github.com/codespell-project/codespell/pull/1422

**Comment by banesullivan** [comment]
FYI: the notebooks are autogenerated from the `.py` files in the `examples` directory, so there is no need to spell check them.

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Simple makefile to simplify repetitive build env management tasks under posix
 
-CODESPELL_DIRS ?= "docs pyvista examples"
-CODESPELL_SKIP ?= "*.pyc,*.txt,*.gif,*.png,*.jpg,*.ply,*.vtk,*.vti"
+CODESPELL_DIRS ?= pyvista/ examples/ tests/
+CODESPELL_SKIP ?= "*.pyc,*.txt,*.gif,*.png,*.jpg,*.ply,*.vtk,*.vti,*.js,*.html,*.doctree,*.ttf,*.woff,*.woff2,*.eot,*.mp4,*.inv,*.pickle"
```
Comment (banesullivan) [inline Makefile]: let's add `.ipynb`

**Comment by GuillaumeFavelier** [comment]
It should be good to go now

**Pull request merged**